cgmanifest.json

```diff
@@ -6726,6 +6736,26 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "influxdb2",
+          "version": "2.4.0",
+          "downloadUrl": "https://github.com/influxdata/influxdb/archive/refs/tags/v2.4.0.tar.gz"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "influx-cli",
```

Comment:
Does the version number of influx-cli need to stay in sync with influxdb2? If so, please update toolkit/scripts/check_entangled_specs.py to enforce this.